modules/ossm-directing-egress-traffic-through-a-gateway-using-istio-apis.adoc

```diff
@@ -0,0 +1,226 @@
+// This procedure is used in the following assembly:
+// * gateways/ossm-directing-outbound-traffic-through-a-gateway
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-directing-egress-traffic-through-a-gateway-using-istio-apis_{context}"]
+= Directing egress traffic through a gateway using Istio APIs
+:context: ossm-directing-egress-traffic-through-a-gateway-using-istio-apis
+
+Follow this procedure to use {istio} APIs to direct outbound HTTP traffic through a gateway that was installed using gateway injection.
+
+.Prerequisites
+
+* You have installed a gateway using gateway injection.
+
+.Procedure
+
+. Run the following command to create a namespace called `curl`:
++
+[source,terminal]
+----
+$ oc create namespace curl
+----
+
+. Depending on the update strategy you're using, enable sidecar injection in the namespace by running the appropriate command(s): 
+
+.. If you are using the `InPlace` update strategy, run the following command:
++
+[source,terminal]
+----
+$ oc label namespace curl istio-injection=enabled
+----
+
+.. If you are using the `RevisionBased` update strategy, run the following commands:
+
+... Find your `<revision-name>` by running the following command:
++
+[source, terminal]
+----
+$ oc get istiorevisions.sailoperator.io
+----
++
+.Example output:
+[source,terminal]
+----
+NAME              TYPE    READY   STATUS    IN USE   VERSION   AGE
+default-v1-23-0   Local   True    Healthy   True    v1.23.0   3m33s
+----
+
+... Run the following command to label the namespace with the revision name to enable sidecar injection:
```

Comment:
Recommend making this and the above step parallel in where you put the "run the following command". Here it's at the beginning, but the previous step has it at the end.

Reply:
done